packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -504,15 +504,15 @@ export class ProfilesUtils {
         }
     }
 
-    private static v1ProfileOptions(): void {
+    private static async v1ProfileOptions(): Promise<void> {
         const v1ProfileErrorMsg = vscode.l10n.t(
             // eslint-disable-next-line max-len
-            "Zowe v1 profiles in use.\nZowe Explorer no longer supports v1 profiles, choose to convert existing profiles to a team configuration or create new."
+            "Zowe V1 profiles in use.\nZowe Explorer no longer supports V1 profiles. Choose to convert existing profiles to a team configuration or create new profiles."
         );
         ZoweLogger.warn(v1ProfileErrorMsg);
-        const createButton = vscode.l10n.t("Create New");
         const convertButton = vscode.l10n.t("Convert Existing Profiles");
-        Gui.infoMessage(v1ProfileErrorMsg, { items: [createButton, convertButton], vsCodeOpts: { modal: true } }).then(async (selection) => {
+        const createButton = vscode.l10n.t("Create New");
+        await Gui.infoMessage(v1ProfileErrorMsg, { items: [convertButton, createButton], vsCodeOpts: { modal: true } }).then(async (selection) => {
```

Comment:
since we made this (`v1ProfileOptions`) function asynchronous, we should probably remove the nested `.then` here and operate on the returning value/selection of the `infoMessage` function.